### ISSUE TITLE
Add `How to get & send data to other sidechains`

### ISSUE THREAD

**Issue by shuse2** [LiskArchive/lisk-legacy-docs#1525]
### Description
- Explain how to create interoperable module which sends CCM to other chains
- Explain how to add logic to interoperable module which receives CCM from other chains

### Additional Information
- Add a new command and a cc command to the hello world app
Requires: https://github.com/LiskHQ/lisk-sdk/issues/7211

**Comment by Tschakki** [comment]
Duplicate of #1678
